front/tests/pages/op-timetable-page-model.ts

```diff
@@ -200,12 +200,13 @@ class OperationalStudiesTimetablePage extends CommonPage {
     const trainCount = await this.timetableTrains.count();
 
     for (let currentTrainIndex = 0; currentTrainIndex < trainCount; currentTrainIndex += 1) {
-      await this.page.waitForLoadState('networkidle');
-      await this.waitForSimulationResults();
-      const trainButton = OperationalStudiesTimetablePage.getTrainButton(
-        this.timetableTrains.nth(currentTrainIndex)
-      );
+      const trainElementHandle = this.timetableTrains.nth(currentTrainIndex);
```

Comment:
Kind of orthogonal to this PR, but it's nicer to use `for (const trainElement of await this.timetableTrains.all())` instead of a counter.